Asynchronous entity world add exception

I am using v1.2.4 from https://www.spigotmc.org/resources/bedwarsproxy.66642. The server version is latest [PandaSpigot](https://github.com/hpfxd/PandaSpigot).
```
[Craft Scheduler Thread - 2751/WARN]: Exception in thread "Craft Scheduler Thread - 2751"
[Craft Scheduler Thread - 2751/WARN]: org.apache.commons.lang.UnhandledException: Plugin BedWarsProxy v${gitVer}1.2.4 generated an exception while executing task 2190681
at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
at org.github.paperspigot.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:23)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: Asynchronous entity world add!
at org.spigotmc.AsyncCatcher.catchOp(AsyncCatcher.java:14)
at net.minecraft.server.v1_8_R3.World.b(World.java:2867)
at net.minecraft.server.v1_8_R3.Chunk.addEntities(Chunk.java:921)
at org.bukkit.craftbukkit.v1_8_R3.chunkio.ChunkIOProvider.callStage2(ChunkIOProvider.java:49)
at org.bukkit.craftbukkit.v1_8_R3.chunkio.ChunkIOProvider.callStage2(ChunkIOProvider.java:18)
at org.bukkit.craftbukkit.v1_8_R3.util.AsynchronousExecutor.skipQueue(AsynchronousExecutor.java:337)
at org.bukkit.craftbukkit.v1_8_R3.util.AsynchronousExecutor.getSkipQueue(AsynchronousExecutor.java:295)
at org.bukkit.craftbukkit.v1_8_R3.chunkio.ChunkIOExecutor.syncChunkLoad(ChunkIOExecutor.java:16)
at net.minecraft.server.v1_8_R3.ChunkProviderServer.getChunkAt(ChunkProviderServer.java:136)
at net.minecraft.server.v1_8_R3.ChunkProviderServer.getChunkAt(ChunkProviderServer.java:118)
at org.bukkit.craftbukkit.v1_8_R3.CraftWorld.getChunkAt(CraftWorld.java:145)
at org.bukkit.craftbukkit.v1_8_R3.CraftWorld.getBlockAt(CraftWorld.java:91)
at org.bukkit.craftbukkit.v1_8_R3.CraftWorld.getBlockAt(CraftWorld.java:592)
at org.bukkit.Location.getBlock(Location.java:87)
at com.andrei1058.spigot.signapi.SignVersion.update(SignVersion.java:23)
at com.andrei1058.spigot.signapi.SignListener.lambda$onTP$1(SignListener.java:89)
at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftTask.run(CraftTask.java:59)
at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:53)
... 4 more
```

--- a/src/main/java/com/andrei1058/spigot/signapi/SignListener.java
+++ b/src/main/java/com/andrei1058/spigot/signapi/SignListener.java
@@ -82,7 +82,7 @@ public class SignListener implements Listener {
         if (e.isCancelled()) return;
         if (e.getTo() == null) return;
         final World world = e.getPlayer().getWorld();
-        Bukkit.getScheduler().runTaskLaterAsynchronously(api.client, () -> {
+        Bukkit.getScheduler().runTaskLater(api.client, () -> {
             for (PacketSign a : api.getSigns()) {
                 if (a.getWorld().equals(e.getPlayer().getWorld().getName())) {
                     if (a.isInRange(e.getTo().getBlockX(), e.getTo().getBlockZ())) {

